.github/workflows/build.yml

```
@@ -9,16 +9,16 @@ jobs:
+      - name: Checkout code
+        uses: actions/checkout@34e114876b0b11c390a56381ad16ebd13914f8d5 # v4
 
-    - name: Build
-      env:
-        GOARCH: ${{ matrix.goarch }}
-        GOOS: ${{ matrix.goos }}
-      run: ./hack/build-go.sh
+      - name: Build
+        env:
+          GOARCH: ${{ matrix.goarch }}
+          GOOS: ${{ matrix.goos }}
```

actionlint (1.7.12)

[error] property "goos" is not defined in object type {go-version: string; goarch: string; os: string}

(expression)